skeleton/gitops-template/components/http/overlays/prod/deployment-patch.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:   
+  annotations:  
+    tad.gitops.set/image: ".spec.template.spec.containers[0].image"
+    tad.gitops.get/image: ".spec.template.spec.containers[0].image"
+    tad.gitops.set/replicas: ".spec.replicas"
+    tad.gitops.get/replicas: ".spec.replicas" 
+  name: ${{ values.name }}
```

Comment:
should the name be `<name>-prod` to match the application yaml?

Reply:
It doesn't have to be different so I left it the same name in each namespace. Easier to handle as all you do it vary the namespace, the deployment name remains the same
ie script can be easier 
``` 
oc get deployment name -n name-development 
oc get deployment name -n name-stage 
oc get deployment name -n name-prod 
```